# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/d3d12/src/sync.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/d3d12/src/sync.rs b/third_party/rust/d3d12/src/sync.rs
--- a/third_party/rust/d3d12/src/sync.rs
+++ b/third_party/rust/d3d12/src/sync.rs
@@ -1,39 +1,39 @@
-use crate::{com::WeakPtr, HRESULT};
-use std::ptr;
-use winapi::um::{d3d12, synchapi, winnt};
-
-#[derive(Copy, Clone)]
-#[repr(transparent)]
-pub struct Event(pub winnt::HANDLE);
-impl Event {
-    pub fn create(manual_reset: bool, initial_state: bool) -> Self {
-        Event(unsafe {
-            synchapi::CreateEventA(
-                ptr::null_mut(),
-                manual_reset as _,
-                initial_state as _,
-                ptr::null(),
-            )
-        })
-    }
-
-    // TODO: return value
-    pub fn wait(&self, timeout_ms: u32) -> u32 {
-        unsafe { synchapi::WaitForSingleObject(self.0, timeout_ms) }
-    }
-}
-
-pub type Fence = WeakPtr<d3d12::ID3D12Fence>;
-impl Fence {
-    pub fn set_event_on_completion(&self, event: Event, value: u64) -> HRESULT {
-        unsafe { self.SetEventOnCompletion(value, event.0) }
-    }
-
-    pub fn get_value(&self) -> u64 {
-        unsafe { self.GetCompletedValue() }
-    }
-
-    pub fn signal(&self, value: u64) -> HRESULT {
-        unsafe { self.Signal(value) }
-    }
-}
+use crate::{com::WeakPtr, HRESULT};
+use std::ptr;
+use winapi::um::{d3d12, synchapi, winnt};
+
+#[derive(Copy, Clone)]
+#[repr(transparent)]
+pub struct Event(pub winnt::HANDLE);
+impl Event {
+    pub fn create(manual_reset: bool, initial_state: bool) -> Self {
+        Event(unsafe {
+            synchapi::CreateEventA(
+                ptr::null_mut(),
+                manual_reset as _,
+                initial_state as _,
+                ptr::null(),
+            )
+        })
+    }
+
+    // TODO: return value
+    pub fn wait(&self, timeout_ms: u32) -> u32 {
+        unsafe { synchapi::WaitForSingleObject(self.0, timeout_ms) }
+    }
+}
+
+pub type Fence = WeakPtr<d3d12::ID3D12Fence>;
+impl Fence {
+    pub fn set_event_on_completion(&self, event: Event, value: u64) -> HRESULT {
+        unsafe { self.SetEventOnCompletion(value, event.0) }
+    }
+
+    pub fn get_value(&self) -> u64 {
+        unsafe { self.GetCompletedValue() }
+    }
+
+    pub fn signal(&self, value: u64) -> HRESULT {
+        unsafe { self.Signal(value) }
+    }
+}